src/modules/dashboard/components/CoinMachine/SynapsKYCDialog/SynapsKYCDialog.css

```diff
@@ -0,0 +1,19 @@
+.content {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 400px;
+}
+
+.synaps iframe {
+  min-height: 400px;
+}
+
+.synaps {
+  min-height: 400px;
+}
```

Comment:
I think something like this will look better:

![Screenshot from 2021-09-03 15-01-35](https://user-images.githubusercontent.com/1193222/132002090-cb773934-0dc2-4d6e-b580-7f76ff842631.png)

This was done by:
```css
.content {
  display: flex;
  justify-content: center;
  align-items: center;
  height: 730px;
}

.synaps {
  width: 100%;
  min-height: 690px;
}
```